app/member-reminders/member-reminders-ui/src/main/kotlin/com/hedvig/android/memberreminders/ui/MemberReminderCards.kt

```diff
@@ -73,6 +74,24 @@ fun MemberReminderCards(
         navigateToConnectPayment = navigateToConnectPayment,
       )
     }
+    AnimatedVisibility(
+      visible = memberReminders.coInsuredInfo != null,
+      enter = cardReminderEnterTransition,
+      exit = cardReminderExitTransition,
+      label = "coInsured animated visibility",
+    ) {
+      ReminderCoInsuredInfo(
+        navigateToContractDetail = {
+          memberReminders.coInsuredInfo
+            ?.coInsuredInfoList
+            ?.firstOrNull()
```

Comment:
Since `coInsuredInfoList` is a NonEmptyList, I think you can just do `first()` without the `orNull` or even just call `coInsuredInfoList.head` and it returns back the non-null version of the first item.

Reply:
Awesome!